opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -16,6 +16,8 @@
 
 from wrapt import ObjectProxy
 
+from yarl import URL
```

Comment:
I think this should be in the same import block as the `wrapt` import based on [PEP 8 import conventions](https://www.python.org/dev/peps/pep-0008/#imports). What do you think?

Reply:
You're right! Running `isort` on the file confirmed this. I was also able to fix the other lint errors using black in vscode.